pyHS100/cli.py

```diff
@@ -19,11 +19,12 @@
 
 @click.group(invoke_without_command=True)
 @click.option('--ip', envvar="PYHS100_IP", required=False)
+@click.option('--host', envvar="PYHS100_HOST", required=False)
```

Comment:
could you add help texts for these options, pointing out that using IP is deprecated and will likely be removed at some point?

Reply:
Okay, sounds good, done.